include/bluetooth/mesh/vnd/silvair_enocean_srv.rst

```diff
@@ -0,0 +1,67 @@
+.. _bt_mesh_silvair_enocean_srv_readme:
+
+Silvair EnOcean Proxy Server
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Silvair EnOcean Proxy Server model integrates an EnOcean switch together with :ref:`bt_mesh_onoff_srv_readme` and :ref:`bt_mesh_lvl_srv_readme` models, enabling dimming and on/off light control.
+It implements the `Silvair EnOcean Switch Mesh Proxy Server`_ specification.
+
+The model initializes the :ref:`bt_enocean_readme` library.
+The EnOcean switch can be automatically commissioned if option :option:`CONFIG_BT_MESH_SILVAIR_ENOCEAN_AUTO_COMMISSION` is set.
+
+The Silvair EnOcean Proxy Server consists of one or two elements.
+Each element handles its own button pair and has its own corresponding :ref:`bt_mesh_onoff_srv_readme` and :ref:`bt_mesh_lvl_srv_readme` models.
+These models do not need any configuration as this is handled by the Silvair EnOcean Proxy Server model.
+However, the :ref:`bt_mesh_onoff_srv_readme` and :ref:`bt_mesh_lvl_srv_readme` models need to be instantiated for each element.
+:c:macro:`BT_MESH_MODEL_SILVAIR_ENOCEAN_BUTTON` is provided to do this, and should be used as presented below:
+
+.. code-block:: c
+
+   struct bt_mesh_silvair_enocean_srv enocean;
+
+   static struct bt_mesh_elem elements[] = {
+	   BT_MESH_ELEM(
+         1,
+		   BT_MESH_MODEL_LIST(BT_MESH_MODEL_SILVAIR_ENOCEAN_BUTTON(&silvair_enocean, 0)),
+		   BT_MESH_MODEL_LIST(BT_MESH_MODEL_SILVAIR_ENOCEAN_SRV(&silvair_enocean))),
+   	BT_MESH_ELEM(
+         2,
+		   BT_MESH_MODEL_LIST(BT_MESH_MODEL_SILVAIR_ENOCEAN_BUTTON(&silvair_enocean, 1)),
+		   BT_MESH_MODEL_NONE),
+   };
+
+The Silvair EnOcean Proxy Server does not require any message handler callbacks.
+
+.. note::
+   The state machine behavior in this implementation of the `Silvair EnOcean Switch Mesh Proxy Server`_ specification differs slightly from the published specification document.
+   As in accordance with the specification, no new Generic Delta Set messages are sent when a button is held down for longer than the delta timeout.
+   The deviation from the specification is that the state machine stays in the Delta ON or Delta OFF state, meaning that a Generic OnOff message is not sent when the button is released, even after the delta times out.
+
+States
+=======
```

Comment:
I just noticed that a few lines under some of the headings are longer than the text. Borderline OCD... 🤪